src/App.css

```diff
@@ -3,15 +3,74 @@
   height: 100vh;
   background-color: #fff;
   display: flex;
+  flex-direction: column;
   align-items: center;
+  justify-content: flex-start;
+}
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Thank you I will definitely do it 😄